extensions/GlobalMessages/tests/GlobalMessages0.data.php

```diff
@@ -0,0 +1,970 @@
+* A short description of the vulnerability's potential security impact
+
+In response to security reports using this form, Fandom will:
+* Prioritize the reproduction and confirmation of any reported vulnerability
+* Identify a reasonable timeline to patch any confirmed vulnerability
+* Not pursue legal action against any reporter who complies with all of the guidelines for testing and reporting security issues",
+// 217 -------------------------------------------------------------------------
+'specialcontact-intro-user-conflict' => "Wikis are community spaces where each and every person needs to work together with others for it to be successful. Conflicts can sometimes arise while working together; the best way to solve them is to have an open and respectful conversation on your community about the issue.
+
+If you need help, first contact your [[Special:ListAdmins|local admins]]. Admins are the local experts on the topic and community, so best to first chat with them further. If the admins are not able to help, or you feel the user is violating Wikia Inc.'s [http://www.wikia.com/Terms_of_Use Terms of Use], you can contact Fandom staff directly [[Special:Contact/general|here]].
+
+Happy editing!",
+// 218 -------------------------------------------------------------------------
+'specialcontact-intro-using-wikia' => "Now that you are a part of the community, you are able to edit and contribute to almost any page across Fandom. For some introductory help, check out our [[Help:Getting_Started|help pages here]]. They are a great place to learn the basics of editing, formatting and much more.
+
+If you have questions about this specific community, start by [[Special:ListAdmins|contacting your local admins]].
+You can also ask general questions to other community members on our [[w:c:community:Special:Forum|community help forum]].
+
+Remember, you can't break a community on Fandom, so don't be afraid to get started. Happy editing!",
+// 219 -------------------------------------------------------------------------
+'specialcontact-label-close-account-confirm' => 'I confirm that I want to disable my Fandom account',
+// 220 -------------------------------------------------------------------------
+'specialcontact-mail' => 'Send to Fandom',
+// 221 -------------------------------------------------------------------------
+'specialcontact-mailsub' => 'Fandom Support',
+// 222 -------------------------------------------------------------------------
+'specialcontact-mailsubcc' => 'Copy of Fandom Contact Mail',
+// 223 -------------------------------------------------------------------------
+'specialcontact-pagetitle' => 'Contact Fandom Support Staff',
+// 224 -------------------------------------------------------------------------
+'specialcontact-secheader-editing' => 'Participating on Fandom',
+// 225 -----------------------------------------------------
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Fandom messages" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)